src/app.js

```diff
@@ -17,6 +17,21 @@ const logger = require('../logs/logger');
 const app = express();
 require('express-async-errors');
 
+// Serve the Swagger JSON file
+// app.get('/swagger.json', (req, res) => {
+//   res.sendFile('swagger-output.json');
+// });
```

Comment:
can be removed

Reply:
Removed

Reply:
commit fcca0e0f2998c05fa9dd61004284939c74df7ee0